more_itertools/more.py

```diff
@@ -2783,19 +2782,27 @@ def _sample_weighted(iterable, k, weights):
 def sample(iterable, k, weights=None):
```

Comment:
I would consider adding a `seed=None` parameter.  Then you can set a seed to get deterministic samples.